renovate.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "schedule": "before 5am every weekday",
+  "extends": [
+    "config:best-practices",
+    ":automergeDisabled",
+    ":gitSignOff"
+  ],
+  "baseBranches": [
```

Comment:
Should we try first only master and selectively upgrade on branches later?

Reply:
we can